### PR TITLE
Yarn cache for @esbuild- and @next-swc packages

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Overall changes
======
Update yarn configuration to prevent the @esbuild- and @next-swc- packages constantly updating

Code changes
======

Enables OS's used by Simorgh devs, GitHub & our deployment pipelines

Testing
======
- [x] Put PR on preview environment to make sure pipeline can execute properly - this will give us confidence that it will work once we merge & put it through the CD pipeline (✅ successfully deployed to preview)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -12,3 +12,7 @@ plugins:
     spec: "https://go.mskelton.dev/yarn-outdated/v4"
 
 yarnPath: .yarn/releases/yarn-4.1.1.cjs
+
+supportedArchitectures:
+  os: [current, darwin, linux]
```
Comment (karinathomasbbc) [inline .yarnrc.yml]: Do we also need to support windows for non-devs?

Comment (amoore108) [reply .yarnrc.yml]: May be worth adding as it has come up before?

**Review by amoore108 — APPROVED** [review]
Looks like a neat approach!

**Comment by karinathomasbbc** [comment]
> Looks like a neat approach!

🤞 it works!

**Review by Isabella-Mitchell — APPROVED** [review]

**Pull request merged**